cpp/src/Slice/Parser.cpp

```diff
@@ -573,21 +583,18 @@ Builtin::kindAsString() const
     return builtinTable[_kind];
 }
 
-const char* Slice::Builtin::builtinTable[] =
-    {
-        "byte",
-        "bool",
-        "short",
-        "int",
-        "long",
-        "float",
-        "double",
-        "string",
-        "Object",
-        "Object*",
-        "LocalObject",
-        "Value"
-    };
+optional<Slice::Builtin::Kind>
```

Comment:
The function above this has the signature:
`Builtin::kindAsString() const`
which #756 corrected (for consistency) to be:
`Slice::Builtin::kindAsString() const`

It would be good to do the same here.

Reply:
Fixed